AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -623,13 +624,13 @@ public async Task CommandGetRelationship(string[] cmd)
                 return;
             }
 
-            string source_twin_id = cmd[1];
-            string relationship_name = cmd[2];
-            string relationship_id = cmd[3];
+            string sourceTwinId = cmd[1];
+            //string relationshipName = cmd[2];
```

Comment:
Why is the relationshipName commented out here?

Reply:
It is not used. The API stopped requiring it in the public preview.

Reply:
I stripped it out and removed all the calling code or screen printing that referred to it.